test/test_freetype.out

```diff
@@ -2,15 +2,15 @@
                 
                 
                 
-***    +***+    
-***+   *****   +
-+**+   *****   +
-+***  +**+**+  *
- ***+ ***+**+ +*
- +**+ *** *** +*
- +**++**+ +**+**
-  ***+**+ +**+**
-  ******   *****
-  +****+   +****
-  +****+   +****
-   ****     ****
```

Comment:
It looks like the rendering is slightly thinner with the new version.. do you think this kind of change is normal/acceptable when upgrading freetype.

Reply:
Yeah, it's intentional on the FT side, I assume.